### DISCUSSION TITLE
Out of index file, log files and actual vectors files, what all is stored on S3?

### DISCUSSION THREAD

**Discussion by NehaTamore-Draup** [milvus-io/milvus#22710] [Q&A and General discussion]
We have installed standalone milvus on EC2 machine. With the required config changes, we can store log files and index files on S3. Yet some other files are stored on the EC2 itself(taking about 1GB for 1M data points of ~200 dimensions). Is it the expected behavior or all the files should be on S3 itself?

**Comment by yhmo** [comment]
Index files and vector files are stored in S3/MinIO.
Log files are stored locally, no need to upload to S3/MinIO.

**Reply by NehaTamore-Draup** [reply]
With some mistake in the config, exactly the opposite is happening.
The vector files are stored locally, whereas the log files and index files are stored in S3.
What could be causing this issue?

**Reply by yhmo** [reply]
In the milvus.yaml, the "common" section, the "storageType" is "minio" or "local"?
```
common:
  # please adjust in embedded Milvus: local
  storageType: minio
```
The "storageType" should be "minio" for MinIO and S3.

**Reply by NehaTamore-Draup** [reply]
[docker-compose (3).txt](https://github.com/milvus-io/milvus/files/10967014/docker-compose.3.txt)
[milvus (3).txt](https://github.com/milvus-io/milvus/files/10967015/milvus.3.txt)

Have tried multiple changes, could you suggest changes here, that will be very helpful!

Thanks in advance
